example/app.py

```diff
@@ -14,13 +14,10 @@
 #)
 #app.config['DEBUG_TB_HOSTS'] = ('127.0.0.1', '::1' )
 app.config['SECRET_KEY'] = 'asd'
-
-# TODO: This can be removed once flask_sqlalchemy 3.0 ships
-app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
```

Comment:
Should we keep this in so it's disabled for folks using SQLAlchemy 2.X?

Reply:
Yes, probably the safer approach to leave it in. I re-added it here and in the test app.